package.json

```diff
@@ -15,7 +15,8 @@
     "@millicast/sdk": "^0.1.33",
     "framer-motion": "^6",
     "react": "^18.2.0",
-    "react-dom": "^18.2.0"
+    "react-dom": "^18.2.0",
+    "react-router-dom": "^6.4.2"
```

Comment:
I don't think we need react-router-dom here

Reply:
Is it not needed for `useLocation()`?